install_config/aggregate_logging.adoc

```diff
@@ -52,38 +52,24 @@ to manage container logs and prevent filling node disks.
 [[pre-deployment-configuration]]
 == Pre-deployment Configuration
 
+. An Ansible playbook is available to deploy aggregated logging. You should
+familiarize yourself with the
+xref:../install_config/install/advanced_install.adoc[advanced installation and
+configuration] section. This provides information for prepairing to use Ansible
+and includes information about configuration. Parameters are added to the
+Ansible inventory file to configure various areas of the EFK stack.
+. Review the xref:../aggregate_logging_sizing.adoc[sizing guidelines]
```

Comment:
full path for the link above?

xref:../install_config/aggregate_logging_sizing.adoc[sizing guidelines]